vignettes/bugzilla_wrapper_showcase.Rmd

```diff
@@ -0,0 +1,93 @@
+---
+title: "Bugzilla Wrapper Showcase"
+output: 
+  html_document:
+    toc: true
+    number_sections: true
+vignette: >
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteIndexEntry{Bugzilla Wrapper Showcase}
+  %\VignetteEncoding{UTF-8}
+---
+# Introduction
+This notebook explains how to fetch bugs from Bugzilla from bugzilla site by download the bugzilla issues and comments using perceval. We will be using three function which can be found in the R folder.
+  ~/kaiaulu/R/download.R: download_bugzilla
+  ~/kaiaulu/R/parser.R: parse_bugzilla, parse_bugzillarest
+
+Note: The perceval has two endpoint for bugzilla, so we have two function to parse bugzilla data with different endpoint. One is the traditional backend, and the other one is the REST API backend for Buzilla 5.0 (or higher) servers. The REST API backend will getting more information of issues creator and issue assignee than the traditional backend. For more information, please check: https://github.com/chaoss/grimoirelab-perceval#bugzilla
+
+At the end of the showcase, we will have json object that is downloaded by perceval from bugzilla site, data tables with the bugzilla issues and comments that parse by using traditional perceval or perceval rest.
+
+# Libraries
+
+Please ensure the following R packages are installed on your computer. 
+
+```{r warning = FALSE, message = FALSE}
+rm(list = ls())
+seed <- 1
+set.seed(seed)
+
+require(kaiaulu)
+require(stringi)
+require(data.table)
+require(jsonlite)
+```
+
+# Project Configuration File (Parameters Needed)
+The parameters necessary for analysis are kept in a project configuration file to ensure reproducibility. In this project, we will using the perceval path which is keep in the  tools.yml in this example to parse the bugzilla data. 
+
+Note: you will need to modify the perceval path in tools.yml to your perceval path, you can find your perceval path by running the following command: 
+  Mac and Ubuntu: which perceval
+  Windows: where perceval
```

Comment:
Let's move this to the troubleshooting section of the wiki. It is fine to explain the parameters being used, but for the more general files that are used everywhere (e.g. what is a project configuration file or how tools.yml works) this is best left on the wiki or the README.md to avoid repeating ourselves everywhere.